### PR TITLE
Add testcase for `NoPhpOpenTagInCodeBlockPhpDirective` with `html+php` directive

### LINKED ISSUE #1345: False positive `NoPhpOpenTagInCodeBlockPhpDirective` rule with `html+php` codeblock
From symfony-docs
https://github.com/symfony/symfony-docs/blob/15465df251c19acff319db5304db7c9471be99e2/introduction/from_flat_php_to_symfony.rst?plain=1#LL32-L36C90

This snippet is reported as a Violation

I think with codeblock `html+php` this is normal to use php open tag and this block may be skipped from this rule 
What do you htink about it ?

### PR DESCRIPTION
**Pull request by MrYamous**
Fixes #1345

### CODE REVIEW

File: tests/UnitTestCase.php [hunk 1]
```diff
@@ -37,8 +37,6 @@ public static function phpCodeBlocks(): array
             $result[] = '.. code-block:: '.$codeBlock;
         }
 
-        $result[] = 'A PHP code block follows::';
```
Comment (OskarStark) [inline tests/UnitTestCase.php:40]: This must be kept

Comment (MrYamous) [reply tests/UnitTestCase.php:40]: I will try another approach

**Comment by MrYamous** [comment]
I think it's better this way
If it suits you I will make a PR for Symfony doc when this fix will be available :)

**Comment by OskarStark** [comment]
Fixed by
* https://github.com/OskarStark/doctor-rst/pull/1348

I merge this to have another test case 👍 Thanks for your work 👍

**Pull request merged**